helm-charts/chatqna/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Add some links for the exposed TGI metrics:
```suggestion
#
# Dashboard for the exposed TGI metrics:
# - https://grafana.com/grafana/dashboards/19831-text-generation-inference-dashboard/
# Metric descriptions:
# - https://github.com/huggingface/text-generation-inference/discussions/1127#discussioncomment-7240527

```

(Did you find better description of them any where else.)

Reply:
I wasn't looking...

Reply:
Sorry, that was not a question but a typo. I meant:  "That bug comment was best TGI metrics description I found"